roles/openshift_ovirt/tasks/build_vm_list.yml

```diff
@@ -70,7 +70,8 @@
       'dns_search': '{{ item["dns_search"] }}',
         {% endif -%}
       {% endif -%}
-      'authorized_ssh_keys': '{{ openshift_ovirt_ssh_key }}'
+      'authorized_ssh_keys': '{{ openshift_ovirt_ssh_key }}',
+      'root_password': '{{ nodes_root_password | default(omit) }}'
```

Comment:
You should use an if/endif block.  This looks like it might set root password to '' or possibly some other value (ansible converts omit into some weird string).

Reply:
@tareqalayan setting root_password is already supported on cloud_init dictionary. Just set it in under the VM profile:

```
#vars.yaml
openshift_ovirt_vm_profile:
  master:
    ....
    cloud_init:
        root_password: foo

```

Reply:
also if you want any changes you'd have to post them to the release-3.11